### PR TITLE
Added descriptions for line bars and markers examples

### PR DESCRIPTION
**Pull request by pdubcali**
## PR Summary
From issue #8921
Added some descriptions for the following:
examples/lines_bars_and_markers/scatter_custom_symbol.py
examples/lines_bars_and_markers/scatter_hist.py
examples/lines_bars_and_markers/scatter_masked.py
examples/lines_bars_and_markers/scatter_star_poly.py
examples/lines_bars_and_markers/scatter_symbol.py

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is PEP 8 compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: examples/lines_bars_and_markers/scatter_masked.py [hunk 1]
```diff
@@ -3,6 +3,8 @@
 Scatter Masked
 ==============
 
+Add mask line in plot.
```
Comment (jklymak) [inline examples/lines_bars_and_markers/scatter_masked.py]: Too enigmatic: "mask some data points, and add a line demarking masked regions"

File: examples/lines_bars_and_markers/scatter_hist.py [hunk 2]
```diff
@@ -3,6 +3,9 @@
 Scatter Hist
 ============
 
+Creates histogram from scatter plot.
+And adds them to the sides of the plot.
```
Comment (jklymak) [inline examples/lines_bars_and_markers/scatter_hist.py]: remove first period and capitalization.

File: examples/lines_bars_and_markers/scatter_custom_symbol.py [hunk 3]
```diff
@@ -3,6 +3,9 @@
 Scatter Custom Symbol
 =====================
 
+Custom symbol in scatter plot.
+This example is creating an ellipse
```
Comment (jklymak) [inline examples/lines_bars_and_markers/scatter_custom_symbol.py]: Creating a custom ellipse symbol in scatter plot.

File: examples/lines_bars_and_markers/scatter_masked.py [hunk 4]
```diff
@@ -3,7 +3,8 @@
 Scatter Masked
 ==============
 
-Add mask line in plot.
+Mask some data points and add a line demarking
+masked regions.
```
Comment (timhoffm) [inline examples/lines_bars_and_markers/scatter_masked.py]: This is short enough to fit on a single line.

**Pull request merged**